### PR TITLE
support for gnome 42

### PR DESCRIPTION
**Pull request by zhangfuwen**
this extension works under gnome 42

### CODE REVIEW

**Comment by holocronweaver** [comment]
@Fmstrat Please merge soon-ish, Ubuntu 22.04 upgrade notifications starting going off last week and it uses Gnome 42, so many more users are going to be affected.

Had to manually install, works like a charm.

**Review by chibani — APPROVED** [review]

**Comment by chibani** [comment]
+1 , it works perfectly on Gnome 42.

**Comment by SlippingGitty** [comment]
please merge this

**Comment by jschiefner** [comment]
Until this is merged you can install the extension from the fork for this PR using the correct `patch-1` branch and then following these instructions https://www.debugpoint.com/manual-installation-gnome-extension/ (download the zip from the fork on github, not the gnome website though)

File: metadata.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
 		"3.36",
 		"3.38",
 		"40",
-		"41'"
+		"41'",
```
Comment (phavekes) [inline metadata.json:14]: There seems to be an extra quote here

**Comment by phavekes** [comment]
Fixed in #88

**Pull request closed without merging**